backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -58,6 +58,14 @@ Spree.ready(function(){
   $('.spree_add_fields').click(function() {
     var target = $(this).data("target");
     var new_table_row = $(target + ' tr:visible:last').clone();
+
+    // id of table row
+    new_table_row.attr("id", "spree_new_"+new_table_row.data("hook"));
+
+    // sort handle
```

Comment:
Maybe reduce the time needed to understand what's happening without having for everyone to also read the code?

```
// Remove sort handle
```

Reply:
ok, then comments like "remove id", "remove handle", "add class"..